riscv/encoding.h

```diff
@@ -56,6 +56,10 @@
 #define SSTATUS_UXL         0x0000000300000000
 #define SSTATUS64_SD        0x8000000000000000
 
+#define SSTATUS_VS_MASK     (SSTATUS_SIE | SSTATUS_SPIE | \
+                             SSTATUS_SPP | SSTATUS_FS | SSTATUS_SUM | \
+                             SSTATUS_MXR | SSTATUS_UXL)
+
```

Comment:
This was not generated from the latest riscv-opcodes. If it was, the hash at the top of this file would have changed, and this SSTATUS_VS_MASK would not be here.

Reply:
Regenerated from latest.